.github/workflows/ci.yml

```diff
@@ -61,11 +61,23 @@ jobs:
 
         const tests = split(ENV["TESTS"], ",")
 
+        # Shorten the error messages so we don't need to care about specific paths, file numbers, etc.
```

Comment:
Just to clarify, this only affects CI, not the end result on the website?

Reply:
Well, it affects a local run of the tests with pkg, too. But in any case, doesn't affect the website. Just got tired of the fixtures changing because of path differences and so on.